<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="View" name="view">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>56</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="xPos">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="yPos">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveToButton">
            <property name="text">
             <string>moveTo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Degree:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rotation">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotateButton">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>rotateBy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Coefficient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="coeff">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::DefaultStepType</enum>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scaleButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>scaleBy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="h1">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Selected figure info</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Radius</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="radiusLabel">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;Malgun Gothic&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Perimeter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="perimeterLabel">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;Malgun Gothic&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Area:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="areaLabel">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;Malgun Gothic&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="rectangleLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="widthLabel">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;Malgun Gothic&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="heightLabel">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;Malgun Gothic&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="canvasView">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1315</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header location="global">view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
